### ISSUE TITLE
Fix IntegrityError upon creation of a new TBItem

### ISSUE THREAD

**Issue by Shriukan33** [transport-nantes/tn_web#309]
Django tries to save new items under already used PKs : 

Examples from prod : 
```
django.db.utils.IntegrityError: duplicate key value violates unique constraint "topicblog_topicblogitem_pkey"
DETAIL:  Key (id)=(12) already exists.
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "topicblog_topicblogitem_pkey"
DETAIL:  Key (id)=(11) already exists.
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "topicblog_topicblogitem_pkey"
DETAIL:  Key (id)=(10) already exists.
```
Dev version doesn't reproduce this behavior. 

Gaelle created TBItems on beta yesterday on the add-storytelling-cf branch (was deployed to show the new CF page) and beside the fact no slug was saved, it used unattributed PKs. 

A quick google search showed me [this StackOverflow question](https://stackoverflow.com/questions/19135161/django-db-utils-integrityerror-duplicate-key-value-violates-unique-constraint) that seems to fit, and points that it might be on the side of PostgreSQL. 

I would try to deploy master on beta first to rule out the Postgres. If the same error raises on beta, then it should be something in the code that went wrong. 
If it works fine on beta, I'd follow the SO topic and investigate from here.

**Comment by Shriukan33** [comment]
Fixed : 

> It works now.  Somehow the sequence got hosed.  I don't know how, but I'm going to guess something to do with copying all those pages from Gaelle.
> But I don't know yet why.
> The clue was this.
> \d topicblog_topicblogitem
> So, hmm, there's a sequence number tracked by the object topicblog_topicblogitem_id_seq
> So first look at the existing objects.
> SELECT id FROM topicblog_topicblogitem ORDER BY id;
> And I notice that the errant and duplicant sequence numbers being proposed are incrementing by one, which is smelling suspiciously like pg and not django.
> Django should have no idea what sequence number comes next.  If it did, we'd be repeating the same number surely.
> So I see that the table suggests that a function called nextval is used to increment it.
> I have no clue what this does, but it sure sounds like a function that provides the next value.
> And I'm on beta, so I'm willing to risk breakage, knowing I can, in principle, restore from snapshot and I'm not going to hurt users.
> So let's look at the counter.
> \d topicblog_topicblogitem_id_seq
> Not very informative.  Sometimes we can get more with a +
> \d+ topicblog_topicblogitem_id_seq
> Not this time...
> But let's see what happens when we call nextval
> Last error I got complained about a duplicate 17.
> ```
> localhost tn_beta_django@tn_beta2_db=> SELECT nextval('topicblog_topicblogitem_id_seq'::regclass);
>  nextval 
> ---------
>       18
> (1 row)
> 
> Time: 1.271 ms
> localhost tn_beta_django@tn_beta2_db=>
> 10 h 16
> So that's promising.
> 10 h 17
> Querying the max existing value, I see it's 39.
> 10 h 17
> So that can be done by hand, no need for special tools to advance by thousands.
> 10 h 18
> localhost tn_beta_django@tn_beta2_db=> SELECT nextval('topicblog_topicblogitem_id_seq'::regclass);
>  nextval 
> ---------
>       19
> (1 row)
> 
> Time: 0.469 ms
> localhost tn_beta_django@tn_beta2_db=> SELECT nextval('topicblog_topicblogitem_id_seq'::regclass);
>  nextval 
> ---------
>       20
> (1 row)
> 
> Time: 0.444 ms
> localhost tn_beta_django@tn_beta2_db=> SELECT nextval('topicblog_topicblogitem_id_seq'::regclass);
>  nextval 
> ---------
>       21
> (1 row)
> 
> Time: 0.483 ms
> localhost tn_beta_django@tn_beta2_db=>
> 
> ...
> ...
> 
> localhost tn_beta_django@tn_beta2_db=> SELECT nextval('topicblog_topicblogitem_id_seq'::regclass);
>  nextval 
> ---------
>       43
> (1 row)
> 
> Time: 0.520 ms
> localhost tn_beta_django@tn_beta2_db=> SELECT nextval('topicblog_topicblogitem_id_seq'::regclass);
>  nextval 
> ---------
>       44
> (1 row)
> 
> Time: 0.564 ms
> localhost tn_beta_django@tn_beta2_db=> SELECT nextval('topicblog_topicblogitem_id_seq'::regclass);
>  nextval 
> ---------
>       45
> (1 row)
> 
> Time: 0.606 ms
> localhost tn_beta_django@tn_beta2_db=>
> ```
> In my boredom, I overshot.
> So I tested tb/admin/new , and it no longer 500s

**Comment by JeffAbrahamson** [comment]
I'm not 100% sure of the root cause, but the issue was that the postgresql sequence number for the table's primary key pointed to a value lower than the maximum value in the db.

The primary key in the table `topicblog_topicblogitem` is managed by the sequence `topicblog_topicblogitem_id_seq`, and the table schema as reported by `\d` says it is advanced by `nextval`.

The current value was (at the time) 17, the maximum value in the table was 39.  So I advanced the sequence manually until it was greater than 39.

```
SELECT nextval('topicblog_topicblogitem_id_seq'::regclass);
```

I'm not entirely clear how this could happen, but I suspect it's because I copied in some records that were erroneously entered on beta using `\COPY`.  I'll need to have a closer look at how that would affect sequence numbers.
